frontend/catalyst/api_extensions/__init__.py

```diff
@@ -16,6 +16,7 @@
 This module is a collection of public API extensions for pragramming with Catalyst frontends.
 """
 
+from catalyst.api_extensions.assertion import catalyst_assert
```

Comment:
I can see why you renamed it, should have noticed in the issue that `catalyst.assert` would violate the restricted keyword 😅 

@josh146 , do you think something like `rt_assert()` would be a good name, or do you have another idea?

Reply:
Alright I talked to Josh and we're going to call this `catalyst.debug_assert()` :) 

It would also be good to move the `assertion.py` module into the `catalyst/debug` folder for better organization, and then import the function `debug_assert()` top-level into the `catalyst` module.

Reply:
should be done now :+1: